src/core/LGraphCanvas.js

```diff
@@ -6436,9 +6436,9 @@ export class LGraphCanvas {
 
         // check il filtering gave no results
         if ((sIn.value || sOut.value)
-                        && ((helper.childNodes.length == 0 && options.show_general_if_none_on_typefilter))
+            && ((helper.childNodes.length == 0 && options.show_general_if_none_on_typefilter))
         ) {
-          filtered_extra = [];
+          let filtered_extra = [];
```

Comment:
If you're mutating but not reassigning, this could be `const`.

Reply:
const and let basically only exist within the innermost { } they're declared in, and the statement that owns that.

So if you declare a const in a for loop header, it'll be valid inside the loop.

Also https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSvJvdQTyA6VnVcyUQsBsgM_AvnG0Ml_tjY8A&s